examples/documents/example.proto

```diff
@@ -13,6 +17,12 @@ message ExampleDocument {
   int64 value2 = 4;
   bytes value_bytes1 = 5;
   bytes ValueCamelCased = 6;
+  bytes value_not_ignored = 7 [
```

Comment:
Isn't it enough to say that all values are included unless explicitly specified?

Reply:
Yes, but because it's technically legal syntax, I wanted to add a test case